src/components/ShowGallows/ShowGallows.css

```diff
@@ -0,0 +1,30 @@
+.container {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    padding: 20px;
+
+    @media (min-width: 576px) {
+        padding: 20px;
+    }
+}
+
+.counter {
+    position: absolute;
+    font-size: 16px;
+    color: #851010;
+    text-align: center;
+
+    @media (min-width: 576px) {
+        font-size: 22px;
+    }
+}
+
+.gallowsImage {
+    height: 100%;
+    width: 100%;
+
+    @media (min-width: 576px) {
+        max-width: 700px;
```

Comment:
Unnecessary media query ( you can move the max-width: 700px to the .gallowsImage directly, without media query)